Ginger/GingerCoreCommon/GeneralLib/ObservableList.cs

```diff
@@ -298,21 +297,21 @@ public new IEnumerator<T> GetEnumerator()
             return base.GetEnumerator();
         }
 
-        public void DoLazyLoadItem(string s)
-        {
-            //option 1 simple string 
-            StringData = s;
+        //public void DoLazyLoadItem(string s)
+        //{
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5848147)